modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -0,0 +1,53 @@
+[[aura-reference-security]]
+= Single Sign-On (SSO)
+:description: SSO allows users to log in to the Aura Console using their company IdP credentials.
+
+label:AuraDB-Enterprise[]
+label:AuraDS-Enterprise[]
+
+Aura Enterprise supports Single Sign-On (SSO) at both the Console level and for accessing Workspace, Bloom and Browser clients directly at the instance level.
+
+[NOTE]
+====
+Accessing Aura with SSO requires:
+ 
+* Authorization Code Flow with PKCE.
+* A publicly accessible Identity Provider (IdP) server.
+====
+
+== Console SSO
+
+Console SSO allows users to log in to the Aura Console using their company IdP credentials and grants link:{neo4j-docs-base-uri}/cypher-manual/current/administration/access-control/built-in-roles#access-control-built-in-roles-public[Public Access privileges] to all instances in the tenant.
```

Comment:
Third person plural here and then second person below and again in _Instance SSO_. Neither is wrong, but it should be consistent. 
_Console SSO allows users to..._ and below:
_Instance SSO allows you to..._ , is it possible to rewrite?